ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.17.6
+    tag: latest
```

Comment:
is that wise? shouldn't it match whatever our prod policy is?

Reply:
same comment as below

Reply:
Repo about the ONS docker images: https://github.com/ONSdigital/dp-concourse-tools